superset/config.py

```diff
@@ -1640,7 +1643,7 @@ class ExtraDynamicQueryFilters(TypedDict, total=False):
     try:
         # pylint: disable=import-error,wildcard-import,unused-wildcard-import
         import superset_config
-        from superset_config import *  # type: ignore
+        from superset_config import *
```

Comment:
For the record the precommit hook complained about this so I had to remove it. Not sure how it made it in given it wouldn't let me commit it.

Reply:
That is a little odd... There used to be multiple linters that would battle... perhaps this is a byproduct of that era or something similar

Reply:
yeah this is a major annoyance - I don't know why it complains locally, but for some weird reason it doesn't pass CI 🤦 Must be mismatching versions, I can take a look sometime next week. In the meantime, just revert that change and do good ol' `git commit --amend --no-verify` or similar